std-bits/base/src/main/java/org/enso/base/statistics/CountMinMax.java

```diff
@@ -0,0 +1,53 @@
+package org.enso.base.statistics;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.function.BiFunction;
+import java.util.stream.Stream;
+
+public class CountMinMax {
+  private static boolean isValid(Object v) {
+    return !(v == null || (v instanceof Double && Double.isNaN((Double)v)));
```

Comment:
A `DoubleStorage`-based column can contain both `Nothing` (null) values and `NaN`s and these two are slightly differing situations. I wonder if count should count both as missing? I guess it is ok, but I think the docstring of the `Count` statistic should explain this nuance.